plugins/modules/vmadm.py

```diff
@@ -361,7 +361,9 @@ def get_vm_prop(module, uuid, prop):
     (rc, stdout, stderr) = module.run_command(cmd)
 
     if rc != 0:
-        module.fail_json(msg=f"Could not perform lookup of {prop} on {uuid}", exception=stderr)
+        module.fail_json(
+            msg=f"Could not perform lookup of {prop} on {uuid}:\n{stderr}", exception=traceback.format_exc()
```

Comment:
`traceback.format_exc()` makes no sense here since there is no exception being handled. Same in all other places in this file.